content/news/2022-09-02-gsoc-final-blog-david-chocholaty.md

```diff
@@ -0,0 +1,685 @@
+title: GSoC 2022 Work Product - Pitch Shift effect and Group delay handling
+authors: David Chocholatý
+status: draft
+tags: TODO(davidchocholaty) add tags
```

Comment:
```suggestion
tags: gsoc, gsoc-2022, 
```
Any other tags that would fit?

Reply:
If I am thinking about it, just the latency offers, but not sure about it.